detectable/src/main/java/com/synopsys/integration/detectable/detectables/sbt/parse/SbtResolutionCacheOptions.java

```diff
@@ -9,17 +9,26 @@
 
 import java.util.List;
 
+import org.jetbrains.annotations.Nullable;
+
 public class SbtResolutionCacheOptions {
+    private final String sbtCommandAdditionalArguments;
     private final List<String> includedConfigurations;
     private final List<String> excludedConfigurations;
     private final int reportDepth;
 
-    public SbtResolutionCacheOptions(List<String> includedConfigurations, List<String> excludedConfigurations, int reportDepth) {
+    public SbtResolutionCacheOptions(@Nullable String sbtCommandAdditionalArguments, List<String> includedConfigurations, List<String> excludedConfigurations, int reportDepth) {
```

Comment:
I don't think we should put it in the ResolutionCacheOptions. It's not actually relevant to resolution caches (that is the approach when we read the report files). 

It's also only used in one place so it could just be passed in (whoever constructs the generator pulls it out of the configFactory with like .createAdditionalSbtArguments() and its passed along.